buildprocess/configureWebpack.js

```diff
@@ -51,66 +50,6 @@ function configureWebpack(
   config.module = config.module || {};
   config.module.rules = config.module.rules || [];
 
-  config.module.rules.push({
-    test: /\.js?$/,
-    include: path.dirname(require.resolve("terriajs-cesium/README.md")),
-    exclude: [
-      // require.resolve("terriajs-cesium/Source/ThirdParty/zip"),
-      // require.resolve("terriajs-cesium/Source/Core/buildModuleUrl"),
-      // require.resolve("terriajs-cesium/Source/Core/TaskProcessor")
-    ],
-    loader: StringReplacePlugin.replace({
-      replacements: [
-        // {
-        //   pattern: /buildModuleUrl\([\'|\"|\`](.*)[\'|\"|\`]\)/gi,
-        //   replacement: function (match, p1, offset, string) {
-        //     let p1_modified = p1.replace(/\\/g, "\\\\");
-        //     return (
-        //       "require(`" +
-        //       cesiumDir.replace(/\\/g, "\\\\") +
-        //       "/Source/" +
-        //       p1_modified +
-        //       "`)"
-        //     );
-        //   }
-        // },
-        {
-          pattern: /Please assign <i>Cesium.Ion.defaultAccessToken<\/i>/g,
-          replacement: function () {
-            return 'Please set "cesiumIonAccessToken" in config.json';
-          }
-        },
-        {
-          pattern: / before making any Cesium API calls/g,
-          replacement: function () {
-            return "";
-          }
-        }
-      ]
-    })
-  });
-
-  // The sprintf module included by Cesium includes a license comment with a big
-  // pile of links, some of which are apparently dodgy and cause Websense to flag
-  // web workers that include the comment as malicious.  So here we munge URLs in
-  // comments so broken security software doesn't consider them links that a user
-  // might actually visit.
-  config.module.rules.push({
-    test: /\.js?$/,
-    include: path.resolve(cesiumDir, "Source", "ThirdParty"),
-    loader: StringReplacePlugin.replace({
-      replacements: [
-        {
-          pattern: /\/\*[\S\s]*?\*\//g, // find multi-line comments
-          replacement: function (match) {
-            // replace http:// and https:// with a spelling-out of it.
-            return match.replace(/(https?):\/\//g, "$1-colon-slashslash ");
-          }
-        }
-      ]
-    })
-  });
-
```

Comment:
The `sprintf` credits this was trying to replace is [no longer a dependency of Cesium](https://github.com/CesiumGS/cesium/issues/9796#issuecomment-922063923).